resources/locales/de/translation.json

```
@@ -1 +1,20 @@
-{}
\ No newline at end of file
+{
+    "notifications": {
+        "customBuckets": {
+            "title": "Manuelle Grenzen",
+            "body": "Die manuellen Grenzen umfassen nicht alle der in der Datentabelle eingegebenen Werte."
+        },
+        "numberBucketsExceedsDataSet": {
+            "title": "Anzahl Buckets",
+            "body": "Die Anzahl Buckets ist grösser als die Anzahl unterschiedlicher Werte in der Datentabelle"
+        },
+        "numberCategoriesOutOfColorScale": {
+            "title": "Farbskala",
+            "body": "Die Anzahl unterschiedlicher Kategorien ist höher als wir mit unserer Hauptfarbskala abdecken können"
```

```suggestion
            "body": "Die Anzahl unterschiedlicher Kategorien ist höher als mit der Hauptfarbskala abdeckt werden kann"
```
I would formulate notification messages always in a general way.